src/fileMapper.ts

```diff
@@ -342,27 +338,27 @@ export class FileMapper implements Disposable {
      * @param key key used for mapping, if undefined the mapping won't be added if it exists
      */
     private isLocalFile(uri: Uri, key?: string): boolean {
```

Comment:
What do you want to do with . and ..?

Reply:
Good question. :) There should be a lot of path.normalzie in here somewhere... Let me try that out!

---
In reply to: [418137588](https://github.com/microsoft/sarif-vscode-extension/pull/276#discussion_r418137588) [](ancestors = 418137588)

Reply:
The "normalizing" of paths goes well beyond this PR. started an email thread and I'll talk to Jeff about how this is being handled in the upcoming new code.

---
In reply to: [418149431](https://github.com/microsoft/sarif-vscode-extension/pull/276#discussion_r418149431) [](ancestors = 418149431,418137588)